input/pages/qdm-to-qicore.md

```diff
@@ -1990,7 +1998,7 @@ diet for the infant; however, such an expression could not reference
 clinical intake and output records to determine if anything other
 than human breast milk was administered to the infant. 
 Summarizing discussion among HL7 workgroups in late 2023, the QI-Core resource best suited
-to retrieve information about enteral intake is Observation (i.e., QI-Core Observation in versions STU4.1.1 and STU5.0, and SimpleObservation in STU6.0). 
+to retrieve information about enteral intake is Observation (i.e., QI-Core Observation in versions STU4.1.1 and STU5.0, and SimpleObservation in STU7.0). 
```

Comment:
should be:

to retrieve information about enteral intake is Observation (i.e., QI-Core Observation in versions STU4.1.1 and STU5.0, and SimpleObservation in **STU 6.0 & STU7.0**).